docs: non bundled example

ifedapoolarewaju opened the pull request:
Inline with [documenting uppy for the transloadit website](https://github.com/transloadit/content/pull/371), I wrote a non-bundled version of the example so it's more concise for new users.

arturi commented:
Okay, trying to process what’s what. So you took the example that we used for development and made two, one using the UMD pre-bundled version and one when using Webpack or something, and bundling yourself?

ifedapoolarewaju commented:
Yes so basically I moved the old example into `bundled-example` and left it as is, and added another example that is more compact and doesn't take much to test and get running. I added it so we can point new users to it from the [website documentation](https://github.com/transloadit/content/pull/371).

kvz commented on examples/cdn-example/index.html:
```diff
@@ -0,0 +1,36 @@
+        target: 'body'
+      })
+      .use(Webcam, {target: Dashboard})
+      .use(Tus10, {endpoint: 'http://master.tus.io/files/', resume: true})
```
http**s**://master.tus.io/files/ would be best 😃

arturi replied:
What about `//master.tus.io/files/`?

arturi commented:
👌
